vault/external_tests/policy/no_default_test.go

```diff
@@ -86,4 +84,90 @@ func TestNoDefaultPolicy(t *testing.T) {
 		t.Fatal(diff)
 	}
 }
-*/
+
+func TestPolicy_NoConfiguredPolicy(t *testing.T) {
+	var err error
+	coreConfig := &vault.CoreConfig{
+		DisableMlock: true,
+		DisableCache: true,
+		Logger:       hclog.NewNullLogger(),
+		CredentialBackends: map[string]logical.Factory{
+			"ldap": ldap.Factory,
+		},
+	}
+
+	cluster := vault.NewTestCluster(t, coreConfig, &vault.TestClusterOptions{
+		HandlerFunc: vaulthttp.Handler,
+	})
+
+	cluster.Start()
+	defer cluster.Cleanup()
+
+	cores := cluster.Cores
+
+	vault.TestWaitActive(t, cores[0].Core)
+
+	client := cores[0].Client
+
+	err = client.Sys().EnableAuthWithOptions("ldap", &api.EnableAuthOptions{
+		Type: "ldap",
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Configure LDAP auth backend
+	secret, err := client.Logical().Write("auth/ldap/config", map[string]interface{}{
+		"url":       "ldap://ldap.forumsys.com",
```

Comment:
Please don't reintroduce the flaky external LDAP service.  Look at what the other LDAP tests are doing now with docker and follow that model.

Reply:
I was main going off with the existing test above, `TestPolicy_NoDefaultPolicy`, with adapted changes specific for this test case. I'll switch both to use the container to test.

Reply:
Ah, because the test was commented out it didn't show up when I searched for "forumsys" while removing usages of the external ldap server.  I guess going by the TODO at the top of the file it can now be enabled?

Reply:
Yeah, uncommented and ran the existing test -- passes as well. Both tests now uses the docker image.